vendor/bats-core/Makefile

```diff
@@ -0,0 +1,33 @@
+# Package details
+export VENDOR ?= bats-core
+export DOWNLOAD_URL ?= bats-core.tar.gz::$(REPO_URL)/archive/v$(VERSION).tar.gz
```

Comment:
why the `bats-core.tar.gz::`?

Reply:
because of sanity check doesn't passed with name of archive `archive/v$(VERSION).tar.gz`, due possible names collision. 
`bats-core.tar.gz::` allow override download tarball name
![image](https://user-images.githubusercontent.com/11632146/47319270-e4dcb080-d656-11e8-9b54-b9342586000b.png)